pkg/controller/kubedirectorcluster/members.go

```diff
@@ -239,6 +240,48 @@ func syncMemberNotifies(
 	wgReady.Wait()
 }
 
+// EZML-960
+// extractStartScriptLog extracts the first ~300 symbols
+// from a passed file.
```

Comment:
I kinda think we should show the tail of the stdout/stderr rather than the head. If the stdout/stderr does exceed the limit we've imposed for the member status here, any important stuff is likely to be just before those streams terminate.

Reply:
Done, now it prints few last lines (the exact quantity for each role could be defined in an app spec)